.github/workflows/tag-release.yml

```diff
@@ -78,26 +78,18 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           RELEASE_NAME: ${{ steps.get_version.outputs.version }}
         run: |
-          # find version tags
-          CURRENT_VERSION="anaconda-$RELEASE_NAME-1"
-          PREVIOUS_VERSION="$(git describe --tags --abbrev=0 --exclude="$CURRENT_VERSION")"
-
-          # print commit details; skip the last = release commit
-          git log \
-            --no-merges \
-            --pretty=format:"- %s (%aL)" \
-            "$PREVIOUS_VERSION..$CURRENT_VERSION~1" \
-              | grep -v "#infra" \
-              | grep -v "build(deps-dev)" \
-              | tee release.txt
+          # get changelog from spec file
+          # - get lines starting with %changelog, up to but excluding first following empty line
+          # - then drop the first two lines (start at 3rd) which are %changelog and *DATE COMMITTER
```

Comment:
I really think that we should stop using `%changelog` in upstream spec file completely. Just generate the content by a script and consume it.

The `%changelog` entry here is not correct anyway because it doesn't have items from downstream rebuilds etc...

Reply:
I agree with the direction of the idea, but it does not apply to this thing at this time.

1. This commit was mainly about establishing a single path for this data, which prevents reinventing the same thing. Once we stop doing `%changelog`, then that will change, but that's the future. And we will still want to collect these strings and propagate to the release page, somehow.

2. Downstream rebuilds are obviously downstream. But this builds the upstream tarball release and the text for its page. Does not apply.

Reply:
I missed the part "generate content by script". Yes, a separate script would be a preferable next step for this.